vagrant/provisioning/roles/httpd/templates/ldap-auth-restricted-solr

```diff
@@ -5,7 +5,7 @@ RewriteEngine On
 #LDAPReferrals Off
 # Perform LDAP lookup for user to check group membership
 AuthLDAPUrl "{{ ldap_url }}/{{ ldap_base }}?{{ ldap_remote_user_attribute | default('sAMAccountName') }}?sub?(objectClass=*)"
-AuthLDAPBindDN "{{ ldap_base | regex_search('(?<=\CN=)(.*?)(?=\,)') }}@{{ ldap_user_domain }}"
+AuthLDAPBindDN "{{ ldap_bind_user | regex_search('(?<=\CN=)(.*?)(?=\,)') }}@{{ ldap_user_domain }}"
```

Comment:
this looks OK for the common case; but the fact is that commas are allowed to be in common names ("CN=Miller, David,CN=Users,DC=armedia,DC=com").  Not sure how you would handle this from a regexp perspective.  Anyway for now I guess this will mostly work.

Reply:
I didn't take it into consideration. The regex searches for the first string between 'CN=' and the first comma to appear.